collector/src/main/java/io/prometheus/jmx/JmxCollector.java

```diff
@@ -26,6 +26,8 @@
 
 public class JmxCollector extends Collector {
     private static final Logger LOGGER = Logger.getLogger(JmxCollector.class.getName());
+    public static final String JMX_URL_KEY = "jmxUrl";
```

Comment:
We're not doing this for any other constant, and I don't think the indirection is adding much